assets/src/components/activities/types.ts

```diff
@@ -124,10 +125,14 @@ export interface HasContent {
  * @param id optional identifier
  * @returns content
  */
-export function makeContent(text: string, id?: string): { id: string; content: RichText } {
+export function makeContent(
+  text: string,
+  id?: string,
+): { id: string; content: RichText; editor: EditorType } {
   return {
     id: id ? id : guid(),
     content: [Model.p(text)],
+    editor: getDefaultEditor(),
```

Comment:
The default editor is set at content-creation time.

So if your default editor is slate, you create something, and then change your default editor... that content item will still be slate.

Not having an editor set defaults to slate.